cpp/util/etcd.cc

```diff
@@ -918,5 +966,17 @@ void EtcdClient::Generic(const string& key, const string& key_space,
                       bind(&EtcdClient::FetchDone, this, etcd_req, _1)));
 }
 
+list<EtcdClient::HostPortPair> SplitHosts(const string& hosts_string) {
+  vector<string> hosts(util::split(hosts_string, ','));
+  CHECK(!hosts.empty());
```

Comment:
Removed that `CHECK`; it's impossible to be more specific here, and `EtcdClient()` does a `CHECK` that the passed in list isn't empty.